logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/StartCallUseCase.kt

```diff
@@ -52,7 +54,20 @@ class StartCallUseCase internal constructor(
             conversationType = conversationType,
             isAudioCbr = isAudioCbr
         )
-        Result.Success
+
+        if (conversationType == ConversationType.ConferenceMls) {
+            // TODO update AVS on current epoch after joining the sub conversation
+            joinSubconversationUseCase(conversationId, "conference").fold(
```

Comment:
maybe we can get rid of this hardcoded text?

Reply:
Yep definitely